task/send-to-webhook-discord/0.1/send-to-webhook-discord.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: send-to-webhook-discord
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Messaging
+    tekton.dev/tags: messaging
+    tekton.dev/displayName: "Send a message to Discord channel"
+    tekton.dev/platforms: "linux/amd64,linux/s390x,linux/ppc64le"
+spec:
+  description: >-
+    These tasks post a message to a Discord channel.
+
+    This task uses Webhooks integration of Discord channel to send a message.
+
+  params:
+  - name: webhook-secret
+    type: string
+    description: secret name of Discord webhook URL
+  - name: message
+    type: string
+    description: plain text message
+  - name: bot-name
+    type: string
+    description: plain text message
+    default: 'Tekton Bot'
+  - name: avatar-url
```

Comment:
Should we add a default Avatar url as well(if in case user doesn't have any specific then default avatar url can be used) ?

Reply:
okay, i will add default message and avatar